gramineproject/gramine
# [Docs] Update sphinx to 5.3

## How to test this PR? 

```
make -C Documentation html man
firefox Documentation/_build/html/index.html
man ./Documentation/_build/man/gramine-manifest.1
```

Also see a version matching this PR at https://readthedocs.org/projects/gramine/builds/


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/1784)
